seccomp.go

```diff
@@ -540,6 +626,27 @@ func NewFilter(defaultAction ScmpAction) (*ScmpFilter, error) {
 	return filter, nil
 }
 
+// SetTsync sets or clears the filter's thread-sync (TSYNC) attribute. When set, this attribute
+// tells the kernel to synchronize all threads of the calling process to the same seccomp filter.
+// When using filters with the seccomp notification action (ActNotify), the TSYNC attribute
+// must be cleared prior to loading the filter. Refer to the seccomp manual page (seccomp(2)) for
+// further details.
+func (f *ScmpFilter) SetTsync(val bool) error {
+	var cval C.uint32_t
+
+	if val == true {
+		cval = 1
+	} else {
+		cval = 0
+	}
+
+	err := f.setFilterAttr(filterAttrTsync, cval)
```

Comment:
From seccomp api level 6, we can use `SECCOMP_FILTER_FLAG_TSYNC_ESRCH` to allow tsync and notify together. I think `SetTsync()` should check `apiLevel`, and if appropriate use `filterAttrTsync` and `filterAttrTsyncEsrch` together.

Linux v5.7 adds support for TSYNC and USER_NOTIF together:
https://github.com/torvalds/linux/commit/51891498f2da78ee64dfad88fa53c9e85fb50abf